doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc

```diff
@@ -20,12 +20,12 @@ The *Reports* window is displayed.
 . Click *Create* and fill in the following fields:
 
 * Mandatory fields:
-** *Inventory data file*: Inventory data file generated by CloudForms. Click *Browse* to select the `tar.gz` file.
-** *Report name*: Name of your report, up to 250 characters
-** *Year-over-year growth rate for new hypervisors*: Number of new hypervisors to be added each year
+** *Inventory data file*: a `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms. Click *Browse* to select the `tar.gz` file.
```

Comment:
I think we can remove 'a `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms. ' 

Just 'Click browse, etc.'

Reply:
Done.